formsfeeder.client-cli/src/main/java/formsfeeder/client/cli/CommandLineClient.java

```diff
@@ -40,12 +42,29 @@ public static void mainline(String[] args, InputStream in, PrintStream out, Prin
 			
 			
 			HostParameters hostParams = cliParameters.hostParameters();
-			FormsFeederClient ffClient = FormsFeederClient.builder()
+			FormsFeederClient.Builder ffClientBuilder = FormsFeederClient.builder()
 					  .machineName(hostParams.hostName())
 					  .port(hostParams.hostPort())
 					  .useSsl(hostParams.useSsl())
-					  .plugin(cliParameters.plugin())
-					  .build();	
+					  .plugin(cliParameters.plugin());
+
+			cliParameters.authParameters().ifPresent(
+					authParameters->{
+						if(authParameters instanceof BasicAuthParameters) {
+							ffClientBuilder.basicAuthentication(((BasicAuthParameters)authParameters).username(), ((BasicAuthParameters)authParameters).password());
+						}
+					}
+			);

```

Comment:
Thanks for adding this in (and adding a test)!  It was an oversight on my part, so much appreciated!